image-builder/image_builder.sh

```diff
@@ -57,12 +57,24 @@ EOT
 exit "${error}"
 }
 
-while getopts "ho:s:" opt
+MAX_IMG_SIZE=2048
+FS_TYPE=${FS_TYPE:-"ext4"}
+ATTEMPT_NUM=0
+
+while getopts "ho:s:f:" opt
 do
 	case "$opt" in
 		h)	usage ;;
 		o)	IMAGE="${OPTARG}" ;;
-		s)	IMG_SIZE="${OPTARG}" ;;
+		s)	IMG_SIZE=${OPTARG}
+                        if [ ${IMG_SIZE} -lt 0 ]; then
+                                die "Image size has to be greater than 0 MB."
+                        fi
+                        if [ ${IMG_SIZE} -gt ${MAX_IMG_SIZE} ]; then
+                                die "Image size should not be greater than 2048 MB."
```

Comment:
Can you change this message to print `$MAX_IMG_SIZE` rather than using a hard-coded size here.